### DISCUSSION TITLE
looking for integration with kubernetes

### DISCUSSION THREAD

**Discussion by kaitaklam** [Keyfactor/ejbca-ce#337] [General]
In this URL https://doc.primekey.com/ejbca/ejbca-integration/integrating-with-third-party-applications/issuing-certificates-to-kubernetes-services-using-cert-manager, it mentioned a github using cert-manager. However the github link (https://github.com/Keyfactor/ejbca-containers/tree/master/deployment-examples/kubernetes/cert-manager) is a deadlink.
Is there a github for integration with kubrnetes. Like cert-manager or

**Comment by primetomas** [comment]
Thanks for pointing out the dead link. You can find an updated cert-manager repo here: https://github.com/Keyfactor/ejbca-cert-manager-issuer

There is also a bunch of Keyfactor Community videos on the Kybernetes topics. For example:
https://www.youtube.com/watch?v=CBxBBHis1is
https://www.youtube.com/watch?v=iz2M7V9Pvs4
